File: tests/rspec/pages/utilities/query_results.rb
class QueryResults < ControlPanelPage

  element :search_field, 'input[name=search]'
  element :search_btn, 'input[type=submit]'
  element :table, 'table'
  elements :rows, 'div.box form table tbody tr'
  elements :first_column, 'table tr td:first-child'

end

File: tests/cypress/cypress/elements/pages/utilities/Query_results.js
import ControlPanel from '../ControlPanel'

class QueryResults extends ControlPanel {
    constructor() {
        super()
        //this.url = ;

        this.selectors = Object.assign(this.selectors, {
        	  "search_field" : 'input[type!=hidden][name=search]',
			  "search_btn": 'input[type=submit]',
			  "table" :'table',
			  "rows": 'div.box form table tbody tr',
			  "first_column" :'table tr td:first-child'

        })

    }

}
export default QueryResults;